app/V1Module/presenters/AssignmentsPresenter.php

```diff
@@ -131,7 +138,7 @@ public function checkDefault() {
    */
   public function actionDefault() {
     $assignments = $this->assignments->findAll();
-    $this->sendSuccessResponse($assignments);
+    $this->sendSuccessResponse(array_map([$this->assignmentViewFactory, "getAssignment"], $assignments));
```

Comment:
Not really fan of this...

Reply:
What exactly?

Reply:
Defining the callback as the array... but its a php thing and not a deal breaker

Reply:
Yeah, that's not my jam either.  We could use `Nette\Utils\Callback` here if you want to.

Reply:
Unfortunately, only way to avoid delegate (meaning method+object as callback) construction as a two-item array is to create a wrapping anonymous function. Possibly more comprehensible, but more verbose.